docs/user/installation_and_configuration.rst

```diff
@@ -725,6 +725,21 @@ After building, extract ``target/geomesa-cassandra-gs-plugin_2.11-$VERSION-insta
 
 Restart GeoServer after the JARs are installed.
 
+Additional Configuration
+------------------------
+
+GeoMesa uses a site xml file to maintain system property configurations. This file can be found
+at ``conf/geomesa-site.xml`` of either the GeoMesa-Accumulo or GeoMesa-Kafka distributions. The default settings
```

Comment:
[picky style] We have not been hyphenating the GeoMesa distributions (e.g. "GeoMesa-Accumulo") elsewhere in the docs, and to be consistent I would not do it here either.